SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -416,6 +425,12 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_NON_AUTH_SERVER_INTERES
 								// Tags: Well known entities, non-auth actors, and tombstone tags
 								GDK_KNOWN_ENTITY_TAG_COMPONENT_ID, ACTOR_TAG_COMPONENT_ID, TOMBSTONE_TAG_COMPONENT_ID,
 
+								// Position, required for Strategy Worker
+								POSITION_COMPONENT_ID,
```

Comment:
Without this, we can't use Spatial position with grid LB for remote entities.

Reply:
We could specialize interest only for the strategy worker though, instead of inserting it into every server worker.

Reply:
Grid LB strategy need to know actor positions to be able to decide `ShouldHaveAuthority`, and if we switch over to using Spatial position, the servers then need interest in Position.